gslib/tests/test_rm.py

```diff
@@ -61,6 +67,22 @@ def _CleanRmUiOutputBeforeChecking(self, stderr):
       stderr = stderr.replace(ui_line, '')
     return stderr
 
+  def _CleanOutputLinesForGcloudStorage(self, stderr):
+    """Remove irrelevant lines from the output lines."""
+    stderr_lines = stderr.splitlines()
+    valid_lines = []
+    strings_to_remove = set(['Removing objects:', 'Removing Buckets:', '  '])
+    for line in stderr_lines:
+      if line in strings_to_remove:
+        continue
+      # Gcloud storage logs a '.' for each resource in non-interactive mode.
+      # This might get added to the start of a valid line.
+      # e.g. .Removing gs://bucket/obj
```

Comment:
Is the period before "Removing" intentional?

Reply:
Yes, it is intentional. I added quotes around the string to make it obvious.